plugins/action_generate_review_assignment_records/plugin.json

```
@@ -3,8 +3,15 @@
   "code": "generateReviewAssignments",
   "name": "Generate Review Assignment Records",
   "description": "Generates review_assignment records for current application",
-  "required_parameters": ["applicationId", "reviewId"],
-  "output_properties": [],
+  "required_parameters": [],
+  "optional_parameters": ["applicationId", "reviewId", "reviewId"],
```

Oh, missed that one:

```suggestion
  "optional_parameters": ["applicationId", "reviewId", "isReview"],
```